src/components/table/use-cases/traces.tsx

```diff
@@ -370,11 +371,19 @@ export default function TracesTable({
       cell: ({ row }) => {
         const traceId = row.getValue("id");
         return traceId && typeof traceId === "string" ? (
-          <DeleteTrace
-            traceId={traceId}
-            isTableAction={true}
-            projectId={projectId}
-          />
+          <>
+            <ExpertScoreButton
```

Comment:
Not sure about the UI of the two buttons here. I would at least add a gap between them.

Reply:
Maybe removing the action column and only using the multi select would be an option.